modules/n1ql/partials/grammar/ddl.ebnf

```diff
@@ -63,7 +63,7 @@ index-with ::= 'WITH' expr
 
 /* tag::create-index[] */
 create-index ::= 'CREATE' 'INDEX' index-name ( 'IF' 'NOT' 'EXISTS' )? 'ON' keyspace-ref
-                 '(' index-key index-order? ( ',' index-key index-order? )* ')'
+                 '(' index-key (('INCLUDE')? 'MISSING')? index-order? ( ',' index-key (('INCLUDE')? 'MISSING')? index-order? )* ')'
```

Comment:
Manual line break &mdash; otherwise the code goes past the edge of the listing block.

```suggestion
                 '(' index-key (('INCLUDE')? 'MISSING')? index-order?
               ( ',' index-key (('INCLUDE')? 'MISSING')? index-order? )* ')'
```